exposing-to-internet/README.md

```diff
@@ -0,0 +1,88 @@
+
+## Northbound Routing With Private Service Connect (PSC)
+
+This sample makes use of GCP's [Private Service Connect](https://cloud.google.com/vpc/docs/private-service-connect) (aka PSC) capability to connect an external load balancer to the Apigee X instance.  Apigee supports PSC for both [northbound](https://cloud.google.com/apigee/docs/api-platform/system-administration/northbound-networking-psc) (ingress) and [southbound](https://cloud.google.com/apigee/docs/api-platform/architecture/southbound-networking-patterns-endpoints) (egress) connectivity.  This sample only deals with use of PSC for northbound connections from the internet.
+
+Each Apigee X instance contains a PSC [Service Attachment](https://cloud.google.com/vpc/docs/about-vpc-hosted-services#service-attachments). Information about the attachment can be found on the [Instances](https://cloud.google.com/apigee/docs/api-platform/system-administration/instances) page in the Apigee management UI, or via the [`organizations.instances.get`](https://cloud.google.com/apigee/docs/reference/apis/apigee/rest/v1/organizations.instances/get) API method.
+
+Customers can connect an external load balancer to this attachment using a [PSC network endpoint group](https://cloud.google.com/load-balancing/docs/negs#psc-neg), or PSC NEG for short.  PSC provides a fully managed option to establish connectivity, which does not involve the use of network bridge VMs.   The high level architecture is depicted in the diagram below:
+
+![Architecture](https://cloud.google.com/static/apigee/docs/api-platform/images/psc-arch.png)
+
+## Prerequisites
+1. [Provision Apigee X](https://cloud.google.com/apigee/docs/api-platform/get-started/provisioning-intro) (you may configure to use internal access routing only)
+2. Make sure the following tools are available in your terminal's `$PATH` (Cloud Shell has these preconfigured)
+    * [gcloud SDK](https://cloud.google.com/sdk/docs/install)
+    * curl
+    * jq
+    * npm
+
+# (QuickStart) Setup using CloudShell
+
+Use the following GCP CloudShell tutorial, and follow the instructions.
+
+[![Open in Cloud Shell](https://gstatic.com/cloudssh/images/open-btn.png)](https://ssh.cloud.google.com/cloudshell/open?cloudshell_git_repo=https://github.com/GoogleCloudPlatform/apigee-samples&cloudshell_git_branch=main&cloudshell_workspace=.&cloudshell_tutorial=exposing-to-internet/docs/cloudshell-tutorial.md)
+
+## Setup instructions
+
+1. Clone the `apigee-samples` repo, and switch the `exposing-to-internet` directory
+
+```bash
+git clone https://github.com/GoogleCloudPlatform/apigee-samples.git
+cd exposing-to-internet
+```
+
+2. Edit `env.sh` and configure the following variables:
```

Comment:
Since its just one variable that too PROJECT_ID

Do we need to use the env.sh? Why not just paste the gcloud command to set the project to the variable?